linera-core/src/chain_worker/state/mod.rs

```diff
@@ -99,6 +100,118 @@ where
         self.chain.chain_id()
     }
 
+    /// Handles a request and applies it to the chain state.
+    #[instrument(skip(self, request))]
```

Comment:
Nit: `skip_all` (but we should trace the request! maybe inside as `tracing::debug!(?request, "handling request")`?)